src/Rule/LatteTemplatesRule.php

```diff
@@ -82,11 +81,8 @@ public function getNodeType(): string
      */
     public function processNode(Node $collectedDataNode, Scope $scope): array
     {
-        $resolvedNodes = $collectedDataNode->get(ResolvedNodeCollector::class);
-        $resolvedNodeFinder = new ResolvedNodeFinder($collectedDataNode);
-
-        $processedFiles = array_unique(array_keys($resolvedNodes));
```

Comment:
Originally we got processed files here. BUt with custom resolvers we need way to pass it explvitly

Reply:
I see, so it is passed same way from all resolvers - new and old. Ok